tox.ini

```
@@ -124,11 +128,30 @@ deps =
     -e testing
 commands =
     mkdir -p .report
-    coverage run --source={[vars]src_path},testing/src/scenario \
-             -m pytest --ignore={[vars]tst_path}smoke -v --tb native {posargs}
+    coverage run --source={[vars]src_path},{[vars]testing_src_path} \
+             -m pytest --ignore={[vars]tst_path}smoke \
+             --ignore={[vars]tst_path}benchmark --ignore={[vars]testing_tst_path}benchmark \
+             -v --tb native \
+             -W 'ignore:Harness is deprecated:PendingDeprecationWarning' {posargs}
     coverage xml -o .report/coverage.xml
     coverage report
 
+[testenv:benchmark]
+description = Run benchmark tests
+passenv =
+    RUN_REAL_PEBBLE_TESTS
+    PEBBLE
+deps =
+    PyYAML==6.*
+    websocket-client==1.*
+    pytest~=7.2
+    pytest-benchmark
```

Maybe request a compatible version rather than any, wdyt?

```suggestion
    pytest-benchmark~=5.0
```